entity-types/uninstrumented-awselasticachememcachedcluster/definition.yml

```diff
@@ -0,0 +1,6 @@
+domain: UNINSTRUMENTED
+type: AWSELASTICACHEMEMCACHEDCLUSTER
```

Comment:
Since the category is `MEMCACHED`, the uninstrumented type should also be `MEMCACHED` because we want it to be agnostic of the infrastructure/cloud.